packages/aws-cdk-lib/aws-synthetics/test/canary.test.ts

```diff
@@ -224,7 +224,7 @@ test('Python runtime can be specified', () => {
 
   // WHEN
   new synthetics.Canary(stack, 'Canary', {
-    runtime: synthetics.Runtime.SYNTHETICS_PYTHON_SELENIUM_2_0,
+    runtime: synthetics.Runtime.SYNTHETICS_PYTHON_SELENIUM_4_0,
```

Comment:
Since syn-python-selenium-2.0 is already deprecated, I've decided to make modifications at this time